backend/index.js

```diff
@@ -121,26 +123,31 @@ app.post('/api/github/token', cors(), (req, res, next) => {
 // verifying token
 function verifyToken(req, res, next) {
   // getting auth header
-  const bearerHeader = req.headers['authorization'];
+  const bearerHeader = req.headers["authorization"];
 
   //check if token is there or not
   if (bearerHeader) {
     //split at the space
-    const bearer = bearerHeader.split(' ');
+    const bearer = bearerHeader.split(" ");
     // get token from splited array
     const bearerToken = bearer[1];
-
+    console.log(bearerToken);
```

Comment:
Please remove all remaining `console.log` that are for debugging only. Thank you.